models/user.go

```diff
@@ -25,6 +26,15 @@ type User struct {
 	Username   string `db:"username" json:"username"`
 }
 
+type Article struct {
+    UnreadRoomResponse `json:"article"`
+}
+
+type UnreadRoomResponse struct {
+    UserId string   `json:"key"`
+    Rooms  []string `json:"pending"`
```

Comment:
Nit: I think these names might be a little confusing. Could we just continue to call this user_id, and call the []Rooms something like unread? (There's probably a better name than unread -- I guess that's a little confusing or ambiguous too.)